doc/user/content/installation/install-on-aws/_index.md

```diff
@@ -29,12 +29,10 @@ modules](https://github.com/MaterializeInc/terraform-aws-materialize) to:
 
 {{< /warning >}}
 
-When operating in AWS, we recommend:
+{{% self-managed/aws-recommended-instances %}}
```

Comment:
Single sourcing into a table (file is below https://github.com/MaterializeInc/materialize/pull/31971/files#diff-e74a60ba2e05c4889d13a158eb8b997a70f692a06c8f5b3f1d0c5d4b17262d33)

https://preview.materialize.com/materialize/31971/self-managed/v25.1/installation/install-on-aws/